chapter-07/final/collectors/load.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 // Load reads the file and returns the list of collectors, and their beloved books, found therein.
-func Load[T comparable](filePath string) (Collectors[T], error) {
+func Load[T collectible](filePath string) (Collectors[T], error) {
```

Comment:
```suggestion
func Load[T any](filePath string) (Collectors[T], error) {
```

Reply:
can't - we need to return a Collectors[T]